pkg/proc/target_group.go

```diff
@@ -12,12 +15,18 @@ import (
 // enabled and the backend supports it, otherwise the group will always
 // contain a single target process.
 type TargetGroup struct {
-	targets  []*Target
-	Selected *Target
+	targets           []*Target
+	Selected          *Target
+	followExecEnabled bool
 
 	RecordingManipulation
 	recman RecordingManipulationInternal
 
+	// StopReason describes the reason why the target process is stopped.
```

Comment:
This is specific to the selected target, so it should be stated explicitly here since this is `TargetGroup`.